crawl-ref/source/player.cc

```diff
@@ -6652,7 +6653,8 @@ void player::splash_with_acid(const actor* evildoer, int acid_strength,
     const int dam = roll_dice(4, acid_strength);
     const int post_res_dam = resist_adjust_damage(&you, BEAM_ACID, dam);
 
-    mpr("You are splashed with acid!");
+    mprf("You are splashed with acid%s",
+         attack_strength_punctuation(post_res_dam).c_str());
     if (post_res_dam > 0)
     {
         mpr(hurt_msg ? hurt_msg : "The acid burns!");
```

Comment:
This message should be removed and "but take no damage" added to the splash message.